repositories/diagnostics.BUILD.bazel

```diff
@@ -93,3 +96,73 @@ ros2_cpp_binary(
         "@ros2_rclcpp//:rclcpp",
     ],
 )
+
+ros2_py_binary(
+    name = "cpu_monitor_node",
+    srcs = ["diagnostic_common_diagnostics/diagnostic_common_diagnostics/cpu_monitor.py"],
+    main = "diagnostic_common_diagnostics/diagnostic_common_diagnostics/cpu_monitor.py",
+    set_up_ament = False,
+    visibility = ["//visibility:public"],
+    deps = [
+        ":py_diagnostic_updater",
+        "@ros2_common_interfaces//:py_diagnostic_msgs",
+        "@ros2_rclpy//:rclpy",
+        requirement("psutil"),
+    ],
+)
+
+# The "hd" in "hd_monitor_node" means "hard drive".  We name it "hd_monitor_node" to stay
+# consistent with the upstream name.
+ros2_py_binary(
+    name = "hd_monitor_node",
+    srcs = ["diagnostic_common_diagnostics/diagnostic_common_diagnostics/hd_monitor.py"],
+    main = "diagnostic_common_diagnostics/diagnostic_common_diagnostics/hd_monitor.py",
+    set_up_ament = False,
```

Comment:
Why do you need `set_up_ament = False` here and below? If not necessary, please remove.